src/v/redpanda/admin_debug.cc

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "cluster/cloud_storage_size_reducer.h"
+#include "cluster/controller.h"
+#include "cluster/shard_table.h"
+#include "redpanda/admin/api-doc/debug.json.hh"
+#include "redpanda/admin_server.h"
+
```

Comment:
Should route registration go into this file too?

Reply:
yeh

Reply:
done